package.json

```diff
@@ -30,14 +30,14 @@
     "url": "https://github.com/ipfs/js-ipfs-api"
   },
   "devDependencies": {
-    "aegir": "^3.2.0",
+    "aegir": "^5.0.1",
     "chai": "^3.5.0",
     "gulp": "^3.9.1",
     "interface-ipfs-core": "^0.4.3",
```

Comment:
greenkeeper already made a PR will merge that afterwards all the things are green